galaxy.yml

```diff
@@ -13,5 +13,5 @@ description: Ansible collection for the Cisco Nexus® Dashboard Fabric Controlle
 license_file: LICENSE 
 tags: [cisco, ndfc, dcnm, nxos, networking, vxlan]
 dependencies:
-  "ansible.netcommon": "*"
+  "ansible.netcommon": ">=2.6.1"
```

Comment:
Have you tested a galaxy install to make sure it installs version `2.6.1` or later?  Just want to make sure this directive works as intended.

Reply:
Yes. When I tested for galaxy install, it selected version 4.1.0 for net common installation